CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+| bug fix |  **a11y:** add parent node check to AriaDescriber ([#19951](https://github.com/angular/components/issues/19951)) ([16a3730](https://github.com/angular/components/commit/16a3730)), closes [#19949](https://github.com/angular/components/issues/19949) |
+| bug fix |  **clipboard:** scroll position changing while copying on some browsers ([#20073](https://github.com/angular/components/issues/20073)) ([a6f1a33](https://github.com/angular/components/commit/a6f1a33)) |
+| bug fix |  **drag-drop:** preserve checked state for grouped radio inputs ([#20237](https://github.com/angular/components/issues/20237)) ([8c336c9](https://github.com/angular/components/commit/8c336c9)), closes [#20236](https://github.com/angular/components/issues/20236) |
+| bug fix |  **layout:** support readonly arrays for public methods ([#20252](https://github.com/angular/components/issues/20252)) ([e3699dc](https://github.com/angular/components/commit/e3699dc)) |
+| bug fix |  **overlay:** ensure OverlayOutsideClickDispatcher listens to contextmenu events ([#20373](https://github.com/angular/components/issues/20373)) ([7a63ad6](https://github.com/angular/components/commit/7a63ad6)) |
+| bug fix |  **overlay:** fix wrong overflow calculation ([#17791](https://github.com/angular/components/issues/17791)) ([d2b499d](https://github.com/angular/components/commit/d2b499d)) |
+| bug fix |  **testing:** emit pointer events on hover and mouseAway ([#20098](https://github.com/angular/components/issues/20098)) ([a5274a9](https://github.com/angular/components/commit/a5274a9)) |
+| feature |  **collections:** extract view repeater strategies from virtualForOf and CDK table. ([#19964](https://github.com/angular/components/issues/19964)) ([3b3b55f](https://github.com/angular/components/commit/3b3b55f)) |
+| feature |  **testing:** add optional excludes to TestElement text method ([#20145](https://github.com/angular/components/issues/20145)) ([8eb6754](https://github.com/angular/components/commit/8eb6754)) |
+| feature |  **testing:** add test harnesses for the datepicker module ([#20219](https://github.com/angular/components/issues/20219)) ([90d6b70](https://github.com/angular/components/commit/90d6b70)) |
+| bug fix |  **testing:** filtering by regex not working ([#20148](https://github.com/angular/components/issues/20148)) ([20e5ea9](https://github.com/angular/components/commit/20e5ea9)) |
+
+
+### material
+
+_Deprecations:_
+
+* **autocomplete:** * `AUTOCOMPLETE_OPTION_HEIGHT` is deprecated, because it isn't being used anymore.
+* `AUTOCOMPLETE_PANEL_HEIGHT` is deprecated, because it isn't being used anymore.
+
+|            |                       |
+| ---------- | --------------------- |
+| feature |  add support for TypeScript 4.0 ([#20368](https://github.com/angular/components/issues/20368)) ([f33210c](https://github.com/angular/components/commit/f33210c)) |
```

Comment:
This shows up twice

Reply:
I intentionally added it twice since it affects both